docs/administration/jmx-rest-management.md

```diff
@@ -1,5 +1,433 @@
 # JMX/REST Management
 
-> 🚧 Work in progress
->
-> https://docs.exoplatform.org/en/6.3/Management.html
\ No newline at end of file
+In this chapter, the following topics are included:
+
+- `Introduction to eXo Platform management` Overall information about managing resources of eXo Platform, JMX and REST interfaces.
+- `Management views of eXo Platform <Management.ManagementViews>` Introduction to the following set of management view types of eXo Platform and their Object Names:
+  - `PortalContainer management view`
+  - `Cache management view`
+  - `Content management view`
+  - `JCR management view`
+  - `Portal management view`
+  - `Forum management view`
+- `Jobs and Job Scheduler` List of the Cron Jobs and the Job Scheduler MBean.
+- `eXo Platform notifications monitoring` A step by step to monitor notifications.
+
+## Introduction to eXo Platform management
+
+Managing resources of eXo Platform is critical for IT operators and system administrators to monitor and supervise the production system.
+eXo Platform can be managed using JMX (Java Management Extension) tools or REST service.
+
+To use JMX, some settings are required. To use REST service, you just need a browser. As you will see later in this chapter, all operations are available in JMX and some of them are available in REST. So use JMX if you need all operations and use REST in some cases, for example, you are on a machine that JMX is not installed, or at remote region where JMX is inaccessible because of security setup.
```

Comment:
```suggestion
To use JMX, some settings are required. To use REST services, you just need a browser. As you will see later in this chapter, all operations are available in JMX and some of them are available in REST. So use JMX if you need all operations and use REST in some cases, for example, you are on a machine that JMX is not installed, or at remote region where JMX is inaccessible because of security setup.
```